### DISCUSSION TITLE
Δηλώσεις για τις παρουσιάσεις προόδου ομαδικής ΓΕ2 Μέρος Δ

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#50] [Show and tell]
Στα παρακάτω σχόλια μπορείτε να κάνετε δήλωση πρόθεσης της προαιρετικής παρουσίασης ομαδικής εργασίας με την ανάρτηση τουλάχιστον μια μέρα πριν την προθεσμία, δλδ Παρασκευή 9 Δεκ.

Οι παρουσιάσεις θα γίνουν με σειρά προτεραιότητας ανάλογα με τον χρόνο δήλωσης παρακάτω.

Σε περίπτωση που ο χρόνος της ΟΣΣ δεν είναι αρκετός τότε κάποιες παρουσιάσεις μπορεί να γίνουν σε επόμενη συμπληρωματική ΟΣΣ

**Comment by afoscolou** [comment]
Θα θέλαμε να κάνουμε δήλωση πρόθεσης της προαιρετικής παρουσίασης προόδου για τα Β' και Γ' μέρη της 2ης εργασίας.

**Reply by sotkot** [reply]
Καλησπέρα σας,

Επισυνάπτω την παρουσίαση της ομαδικής μας εργασίας σε pdf:
[SDY60_2ERG_ANDREADAKIS_KOTROTSIOS_MIOGLOU_FOSCOLOU_part_D.pdf](https://github.com/sdy60/syllabus/files/10196696/SDY60_2ERG_ANDREADAKIS_KOTROTSIOS_MIOGLOU_FOSCOLOU_part_D.pdf)

**Comment by vkaragia** [comment]
Καλημέρα σας , σας υποβάλλω συνημμένα την παρουσίαση της ομαδικής μας εργασίας σε αρχείο pdf:
[PRES GE2_1 KAM_2KAR_TSA.pdf](https://github.com/sdy60/syllabus/files/10199332/PRES.GE2_1.KAM_2KAR_TSA.pdf)
